### PR TITLE
Added UI_APPEARANCE_SELECTOR to bubble view layer properties

### PR DESCRIPTION
**Pull request by DevMoz**
Enabled border width and border color of bubble view to be set via UIAppearance proxy.

### CODE REVIEW

**Comment by kmahal** [comment]
@DevMoz please see how we set `bubbleViewColor` internally in `ATLMessageCollectionViewCell` and follow a similar implementation, including default set up in `lyr_commonInit`.  Be sure to also add tests in `ATLMessageCollectionViewCellTest` for these 2 new properties as well.

File: Code/Views/ATLMessageCollectionViewCell.h [hunk 1]
```diff
@@ -72,6 +72,16 @@ extern CGFloat const ATLAvatarImageTailPadding;
 @property (nonatomic) CGFloat bubbleViewCornerRadius UI_APPEARANCE_SELECTOR;
 
 /**
+ @abstract The border color of the bubble view displayed in the cell.
+ */
+@property (nonatomic, retain) UIColor *borderColor UI_APPEARANCE_SELECTOR;
```
Comment (kmahal) [inline Code/Views/ATLMessageCollectionViewCell.h]: remove `retain`

File: Code/Views/ATLMessageCollectionViewCell.m [hunk 2]
```diff
@@ -395,6 +395,16 @@ - (void)setBubbleViewCornerRadius:(CGFloat)bubbleViewCornerRadius
     self.bubbleView.layer.cornerRadius = bubbleViewCornerRadius;
 }
 
+- (void)setBorderWidth:(double )borderWidth
```
Comment (blakewatters) [inline Code/Views/ATLMessageCollectionViewCell.m]: This should be a `CGFloat`

File: Code/Views/ATLMessageCollectionViewCell.h [hunk 3]
```diff
@@ -72,6 +72,16 @@ extern CGFloat const ATLAvatarImageTailPadding;
...
+
+/**
+ @abstract The border width of the bubble view displayed in the cell.
+ */
+@property (nonatomic) double borderWidth UI_APPEARANCE_SELECTOR;
```
Comment (blakewatters) [inline Code/Views/ATLMessageCollectionViewCell.h]: This should be a `CGFloat`

**Comment by DevMoz** [comment]
Thank you for your feedback. I've tried to follow the implementation for `bubbleViewColor`. This is my very first contribution to an open source project. Sorry, it may take me a bit to figure out how to add the tests. I can close this out if you prefer.

**Comment by kmahal** [comment]
@DevMoz I'll close this one out, but we can re-open it if the PR is updated.

**Pull request closed without merging**